vaultaire-collector-ceilometer.cabal

```diff
@@ -1,16 +1,25 @@
 name:                vaultaire-collector-ceilometer
-version:             0.2.3
-synopsis:            Vaultaire Collector for Ceilometer
+version:             0.4.0
```

Comment:
Because two sets of changes, and I accidentally bumped for each set, will de-bump